modules/openapi-generator/src/test/java/org/openapitools/codegen/java/JavaClientCodegenTest.java

```diff
@@ -1899,38 +1900,6 @@ public void testMicroprofileGenerateCorrectJsonbCreator_issue12622() throws Exce
                 .containsWithNameAndAttributes("JsonbProperty", ImmutableMap.of("value", "\"c\""));
     }
 
-    @Test
-    public void testMicroprofileGenerateCorrectJacksonGenerator_issue18336() throws Exception {
```

Comment:
@wing328 why this test was removed ?

Reply:
thanks for pointing it out. i missed that during review

@jpfinne can you please take a look?

Reply:
This test did not exist in 7.5.0.

So most probably a merge issue.

Reply:
you did remove it in your PR: https://github.com/OpenAPITools/openapi-generator/pull/18405/files#diff-73bab52f5d7424b5d166fb81aa39bfe47a4019ad4b9eb207b05a01c449d0c9b5L1902 so I don't think it's a merge issue.

Reply:
A manual merge from master to my branch, with a manual handling of the conflicts.

i.e. a mistake

Reply:
that's ok. it happened before.

can you please file a PR to restore those changes (deleted as part of the merge from master)?

Reply:
> Btw: I find it hard to justifiy the time spent on this project when the commit are not done in my name

what do you mean?

the commit correctly shows you as the co-author:

```
commit 2a1b4f90df4974a497fd0fe5a545c09007c12688
Author: William Cheng <wing328hk@gmail.com>
Date:   Wed May 1 18:10:35 2024 +0800

    Add new option `allArgConstructor` for java client, spring generators (#18538)

    * allArgConstructor for java

    * Remove leftover from builder pattern branch

    * Rename generateConstructorWithAllArgs and use x- in vendorExtensions

    * Test issue #18340

    * Add evidences for allVars issue (#18340)

    * remove eol

    * update doc

    ---------

    Co-authored-by: jpfinne <jeanpaul@finne.be>

```